rotkehlchen/tests/unit/decoders/test_defisaver.py

```diff
@@ -93,3 +101,97 @@ def test_deactivate_sub(ethereum_inquirer, ethereum_accounts):
         ),
     ]
     assert events == expected_events
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('ethereum_accounts', [['0xb4757065B2C5EEA193CC393bafF0D858987211FC']])
+def test_defisaver_fl_action_v1_0_3(ethereum_inquirer, ethereum_accounts):
+    tx_hash = deserialize_evm_tx_hash('0x54b1df5ae9ea772d195c9f4fe182be12fd7989e1c7342d4bfd9539c68c1da0e0')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(evm_inquirer=ethereum_inquirer, tx_hash=tx_hash)
+    timestamp = TimestampMS(1713039743000)
+    flashloan_amount = FVal('31.326004217665875683')
+    expected_events = [
+        EvmEvent(
+            tx_hash=tx_hash,
+            sequence_index=3,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.RECEIVE,
+            event_subtype=HistoryEventSubType.GENERATE_DEBT,
+            asset=A_WETH.resolve_to_evm_token(),
+            balance=Balance(amount=flashloan_amount),
+            location_label=ethereum_accounts[0],
+            notes=f'Executed flashloan of {flashloan_amount} WETH via DefiSaver',
+            counterparty=CPT_DEFISAVER,
+            address=FL_ACTION_V1_0_3,
+        ), EvmEvent(
+            tx_hash=tx_hash,
+            sequence_index=35,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.PAYBACK_DEBT,
+            asset=A_WETH.resolve_to_evm_token(),
+            balance=Balance(amount=flashloan_amount),
+            location_label=ethereum_accounts[0],
+            notes=f'Repaid flashloan of {flashloan_amount} WETH via DefiSaver',
+            counterparty=CPT_DEFISAVER,
+            address=FL_ACTION_V1_0_3,
+        ),
+    ]
+    assert set(expected_events).issubset(set(events))
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('ethereum_accounts', [['0x72A559b0A41368ADC80Fbe77702a9A3CbB7C5156']])
+def test_defisaver_fl_action_v1_0_3_bis(ethereum_inquirer, ethereum_accounts):
+    tx_hash = deserialize_evm_tx_hash('0xa3a8946f60d9718b4299dcc8aad0d9bc1340593300b630d70f3da21ca2f26893')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(evm_inquirer=ethereum_inquirer, tx_hash=tx_hash)
+    user_safe_proxy = '0x72A559b0A41368ADC80Fbe77702a9A3CbB7C5156'
```

Comment:
this one is ethereum_accounts[0]

Reply:
yes, I added it only for clarity and discussions